imports/stylesheets/variables.import.styl

```diff
@@ -79,6 +79,7 @@ $incident-type-colors = {
   'date': lighten($positive, 50%)
   'case': lighten($highlight, 60%)
   'uncertain': darken($l-gray, 5%)
+  'disease': #f1d9eb
```

Comment:
Looks like instead of explicitly setting the disease property to the hex value and because its the same as the case color (`lighten($highlight, 60%)`), we could use a variable. Maybe `$highlight-light`?